grafana-plugin/package.json

```diff
@@ -17,16 +17,12 @@
     "@grafana/toolkit": "latest",
     "@grafana/ui": "latest",
     "@grafana/runtime": "latest",
-    "@testing-library/jest-dom": "5.4.0",
-    "@testing-library/react": "^10.0.2",
-    "@types/base-64": "^1.0.0",
     "@types/lodash": "latest"
   },
+  "resolutions": {
+  "rxjs": "7.3.0"
+  },
   "engines": {
     "node": ">=14"
-  },
-  "dependencies": {
-    "@grafana/runtime": "^8.3.2",
-    "node-sass": "^7.0.1"
```

Comment:
`"node-sass": "^7.0.1"` is necessary for compiling grafana-plugin on m1 macos 12.3. 
Any reason to remove it?

Reply:
This has been removed from the newly generated file of grafana backend plugin